pkg/network/state.go

```diff
@@ -252,9 +253,11 @@ func (ns *networkState) StoreClosedConnection(conn ConnectionStats) {
 			prev.LastUpdateEpoch = conn.LastUpdateEpoch
 			client.closedConnections[string(key)] = prev
 		} else if len(client.closedConnections) >= ns.maxClosedConns {
+			ns.telemetry.connsOpened++
 			ns.telemetry.closedConnDropped++
 			continue
 		} else {
+			ns.telemetry.connsOpened++
```

Comment:
This is a little confusing to me - why are we using closed connection events to calculate: # opened conns?

Reply:
This means our telemetry will miss all long lived connections, no?

Reply:
AFAIK this function is called only when the probe is reporting a closed connection that was not included in the other statistics that are fetched on an interval. A common case would be a short-lived connection that was opened and closed between fetches of the ePBF maps. @leeavital correct me if I'm wrong here.

Reply:
@sunhay Does this make sense? https://github.com/DataDog/datadog-agent/blob/fc5d622a4ef68240d11d92688aa7f5deef83541b/pkg/network/state.go#L425 is the line that will capture all long-lived connections.

Reply:
👀

Reply:
So I think this might double (or more) count as `createStatsForKey` is also called for closed connections as well as for each client.

If it helps we can jump on a quick call later today and I can give an overview of `state.go` and some of the subtleties around how it works?